apps/saleor-app-checkout/pages/api/webhooks/mollie.ts

```diff
@@ -31,6 +32,12 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   if (paymentError) {
     console.error(paymentError);
+
+    if (paymentError instanceof MissingPaymentProviderSettingsError) {
+      res.status(500).json({ error: paymentError.message });
```

Comment:
Why are we using

`res.status(500)` here and

```js
return Response.InternalServerError({ error: error.message });
```

in other places? 🤔

Reply:
Because in some places we're using retes.